pmaw/PushshiftAPI.py

```diff
@@ -32,6 +34,8 @@ def search_submission_comment_ids(self, ids, **kwargs):
             mem_safe (boolean, optional) - If True, stores responses in cache during operation, defaults to False
             safe_exit (boolean, optional) - If True, will safely exit if interrupted by storing current responses and requests in the cache. Will also load previous requests / responses if found in cache, defaults to False
             cache_dir (str, optional) - An absolute or relative folder path to cache responses in when mem_safe or safe_exit is enabled
+            sort_var (str, optional) - parameter, used in Pushshift API, to sort values
+            check_totals (bool, optional) - if to check total counts before returning results. API stopped returning total counts as of 2022-12-17
```

Comment:
I was reading the reddit post linked, it looks like we can add `&track_total_hits=true` to ensure the total counts are now accurate